app/controllers/concerns/api_rescuable.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module ApiRescuable
+  extend ActiveSupport::Concern
+
+  included do
+    rescue_from ActiveRecord::RecordNotFound, with: :handle_record_not_found
+    rescue_from ActiveRecord::RecordInvalid, with: :handle_validation_error
+    rescue_from ActiveRecord::RecordNotUnique, with: :handle_record_not_unique
+    rescue_from ActionController::ParameterMissing, with: :handle_api_error
+  end
+
+  private
+
+    def handle_validation_error(exception)
+      log_exception(exception)
+      respond_with_error(exception.record.errors_to_sentence)
+    end
+
+    def handle_record_not_found(exception)
+      log_exception(exception)
+      respond_with_error(exception.message, :not_found)
+    end
+
+    def handle_record_not_unique(exception)
+      log_exception(exception)
+      respond_with_error(exception.record.errors_to_sentence)
+    end
+
+    def handle_api_error(exception)
+      log_exception(exception)
+      respond_with_error(exception.record.errors_to_sentence, :internal_server_error)
+    end
+
+    def log_exception(exception)
+      Rails.logger.info exception.class.to_s
+      Rails.logger.info exception.to_s
+      Rails.logger.info exception.backtrace.join("\n")
+    end
```

Comment:
Move this to a `Loggable` concern.

Reply:
@rishabhanand12 Why is this comment resolved even when changes haven't been made? Why's this `log_exception` still here?